### DISCUSSION TITLE
Need help troubleshooting "Taskbar height and icon size" mod

### DISCUSSION THREAD

**Discussion by oroboros** [ramensoftware/windhawk-mods#276] [Q&A]
When I try to enable it, the little task window appears for a few seconds, and when it disappears, my taskbar shifts some 20px down creating a "blank" space between the taskbar and the maximized window. I tried tweaking with the values, but I can't get rid of the "blank" space unless I say the height is very minimal (which then hides my taskbar completely). Any suggestions?

**Comment by m417z** [comment]
@oroboros moved to windhawk-mods since it's about a specific mod.

I encountered this problem in the past, but I wasn't able to consistently reproduce it, and I'm not sure how to fix it. A workaround that always worked for me it to restart explorer. Have you tried that?

**Comment by oroboros** [comment]
I did, but that didn't solve the issue.

**Reply by m417z** [reply]
Are you using a tablet mode? See this issue: https://github.com/ramensoftware/windhawk-mods/issues/184
If not, can you post a screenshot of how it looks, and your Windows version?
